docker/app/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18-buster
+
+WORKDIR /app/
+
+COPY . .
+
+RUN npm install
```

Comment:
hi @Rilord will be possible to include the default installation SQL file to run automatically in the container initial setup? also see the comment below (thanks!)